### ISSUE TITLE
SubDownloader Error: "Protocol \"\" is unknown"

### ISSUE THREAD

**Issue by AndydeCleyre** [hotvic/QSubber#1]
Lately, subtitles fail to download, with the error in the subject. The output may indicate a parsing mistake.

One example query that leads to this problem is, by name:

`Name`: Travelers
`Season`: 1
`Episode`: 1

```
"Travelers.2016.S01E01.HDTV.x264-KILLERS.srt" : ""
SubDownloader Error: "Protocol \"\" is unknown"
```

**Comment by vaurelios** [comment]
> The output may indicate a parsing mistake.

Yeah! it's a parsing problem, my implementation of XML parser is a bit buggy I'm thinking in rewrite it, in mean time you can use the -gtk variant as it uses other parser.

**Comment by vaurelios** [comment]
I'm moving QSubber to the new REST API, so this problem should be solved along.

[rest-api](/vaurelios/QSubber/tree/rest-api)
